src/components/controls/styles.js

```diff
@@ -18,7 +18,7 @@ export const ControlsContainer = styled.div`
   flex-shrink: 1;
   flex-basis: auto;
   align-self: auto;
-  padding: 50px 20px 20px 20px;
+  padding: 0 20px 20px 20px;
```

Comment:
The original for this setting was `0px` so would prefer if it was reverted to that